cloud-formation/cfn.yaml

```diff
@@ -101,7 +101,9 @@ Resources:
             Statement:
               - Effect: Allow
                 Action: s3:GetObject
-                Resource: !Sub arn:aws:s3:::membership-private/${Stage}/support-frontend.private.conf
+                Resource:
+                - !Sub arn:aws:s3:::membership-private/${Stage}/support-frontend.private.conf
+                - !Sub arn:aws:s3:::gu-zuora-catalog/PROD/Zuora-${Stage}/catalog.json
```

Comment:
Should `PROD` be ${stage} here also?

Reply:
No we need PROD for all environments there weirdly.

Reply:
it's not that weird, actually the first PROD says it's the output from the PROD catalog lambda, and that lambda is set up to fetch from all environments.  The second one is the PROD/UAT/DEV backend for zuora.